code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -100,6 +103,21 @@
 					return
 				message = "<B>[src]</B> [input]"
 
+		if ("cutthroat", "cutthroats")
+			if (!src.restrained() && !src.r_hand)
+				var/mob/M = null
+				m_type = 1
+				if (param)
+					for (var/mob/A in view(src))
+						if (param == A.name)
+							M = A
+							break
+					if (M != src && M != null)
+						var/genderType = gender == MALE ? "his" : "her"
+						message = "<B>[src]</B> points [genderType] finger at [M] and then draws a line across [genderType] throat."

```

Comment:
I'll test it to see if it does work but the reasoning for that was being told the second \his would pick on [M]'s gender.